doc/docs/master/how-tos/mount-volume.md

```diff
@@ -0,0 +1,159 @@
+# Mount a Volume in a Pipeline
+
+You may have a local folder that you want your pipeline to write
+files to. You can mount that folder as a volume in Kubernetes
+and make it available in your pipeline worker by using the
+`pod_patch` pipeline parameter.
+The `pod_patch` parameter takes a string that specifies the changes
+that you want to add to your existing manifest. To create
+a patch, you need to generate a diff of the original ReplicationController
+and the one with your changes. You can use one of the online JSON patch
+utilities, such as [JSON Patch Generator](https://extendsclass.com/json-patch.html)
+to create a diff. A diff for mounting a volume might look like this:
+
+```bash
+[
+ {
+  "op": "add",
+  "path": "/volumes/-",
+  "value": {
+   "name": "task-pv-storage",
+   "persistentVolumeClaim": {
+    "claimName": "task-pv-claim"
+   }
+  }
+ },
+ {
+  "op": "add",
+  "path": "/containers/0/volumeMounts/-",
+  "value": {
+   "mountPath": "/data",
+   "name": "task-pv-volume"
+  }
+ }
+]
+```
+
+This output needs to be converted into a one-liner and added to the
+pipeline spec.
+
+We will use the [OpenCV example](../getting_started/beginner_tutorial/).
+to demonstrate this functionality.
+
+To mount a volume, complete the following steps:
+
+1. Create a PersistentVolume and a PersistentVolumeClaim as
+described in [Configure a Pod to Use a PersistentVolume for Storage](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/). Modify `mountPath` and `path` as needed.
+
+   For testing purposes, you might want to add an `index.html`
+   file as described in [Create an index.html file](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-an-index-html-file-on-your-node).
+
+1. Get the ReplicaController (RC) manifest from your pipeline:
+
+   ```bash
+   kubectl get rc <rc-pipeline> -o json > <filename>.yaml
+   ```
+
+   **Example:**
+
+   ```bash
+   kubectl get rc pipeline-edges-v7 -o json > test-rc.yaml
+   ```
+
+1. Open the generated RC manifest for editing.
+1. Under `spec`, find the `volumeMounts` section.
+1. Add your volume in the list of mounts. 
+
+   **Example:**
+
+   ```bash
+   {
+        "mountPath": "/data",
+        "name": "task-pv-storage"
+   }
+   ```
+
+   `mountPath` is where your volume will be mounted inside of the
+   container.
+
+1. Find the `volumes` section:
+1. Add the information about the volume.
+
+   **Example:**
+
+   ```bash
+   {
+        "name": "task-pv-storage",
+        "persistentVolumeClaim": {
+            "claimName": "task-pv-claim"
+        }
+   }
+   ```
+
+   In this section, you need to specify the PersistentVolumeClaim you have
+   created in Step 1.
+
+1. Save these changes to a new file.
+1. Copy the contents of the original RC to clipboard:
+
+   **Example:**
+
+   ```bash
+   pbcopy < test.yaml
+   ```
```

Comment:
This will only work on mac. Since the way of doing this is so platform-specific, but fairly straight-forward, I'd just remove the example.

Reply:
I believe the majority of our users are on mac. I'll add a note that they can use any tooling. Also, it it s listed as an example.